server/src/endpoint/emails/send-email-verification-email.ts

```diff
@@ -20,14 +22,33 @@ const sendEmailVerificationEmail: RequestHandler<
     return validationBadRequest(bodyValidationResult.errors);
   }
 
-  console.log(body.email);
   const verificationCode = generateVerificationCode(
     body.email,
     config.verificationCodeLength
   );
 
+  // const privateKeyFile : String = process.env.PGP_PRIVATE_KEY;
+  const privateKeyArmored = fs.readFileSync("./info.asc", "utf-8");
+
+  const unsignedMessage = await openpgp.createCleartextMessage({
+    text: body.email
+  });
+
+  const privateKey = await openpgp.decryptKey({
+    privateKey: await openpgp.readKey({ armoredKey: privateKeyArmored }),
+    passphrase: process.env.PGP_PRIVATE_KEY_PASSWORD
+  })
+
+  const signature = await openpgp.sign({
+    message: unsignedMessage,
+    privateKeys: privateKey 
+  });
+
+  console.log(signature);
```

Comment:
```suggestion
```
I think this signature is not being used anywhere, is this intended? Shouldn't it be a property in e.g. `sendEmailVerificationEmail`?